pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -755,14 +755,16 @@ protected void internalCreatePartitionedTopic(AsyncResponse asyncResponse, int n
         return future;
     }
 
-    protected void resumeAsyncResponseExceptionally(AsyncResponse asyncResponse, Throwable throwable) {
-        if (throwable instanceof WebApplicationException) {
-            asyncResponse.resume(throwable);
-        } else if (throwable instanceof BrokerServiceException.NotAllowedException) {
-            asyncResponse.resume(new RestException(Status.CONFLICT, throwable));
+    protected static Void resumeAsyncResponseExceptionally(AsyncResponse asyncResponse, Throwable exception) {
```

Comment:
Why return the `Void` type? It looks meaningless.

Reply:
Could we use it in ``exceptionally`` to avoid unnecessary ``return null``?

Use ``Avoid`` return type
```java
//....
.exceptionally(ex -> {
     return handleCommonRestAsyncException(asyncResponse, ex);
})
```
Don't use ``Avoid`` return type
```java
//....
.exceptionally(ex -> {
     handleCommonRestAsyncException(asyncResponse, ex);
     return null;
})
```